CHANGELOG.md

```diff
@@ -29,6 +30,8 @@ CHANGELOG
   - RDMA core: ``rdma-core-31.2amzn`` (from rdma-core-31.amzn0)
   - Libfabric: ``libfabric-1.11.1amzn1.0`` (from libfabric-1.11.1amzn1.1)
   - Open MPI: ``openmpi40-aws-4.0.5`` (no change)
+- Change the default of EBS IOPS to be 3000 if the volume type is `gp3`, else it is still 100. Change the default of 
```

Comment:
I think the default iops for gp3 is not a change since gp3 was not even supported. Probably we can just remove and leave to bug fixes:
Set the default EBS volume size to 500 GiB when volume type is `st1` or `sc1`

Reply:
Done!